packages/data/addon/models/metadata/column.ts

```diff
@@ -9,12 +9,22 @@ import KegService from '../../services/keg';
 import Table from './table';
 
 export type ColumnType = 'ref' | 'formula' | 'field';
+export interface ColumnMetadata {
```

Comment:
This is kind of unclear to me. The name makes it sound like this is what we expect a column metadata to act/look like, but in our usage it appears to be the shape of the payload needed to construct a column

Reply:
True, we probablly should use constructors/factories to build models and use interfaces to reflect public constracts.